src/App/Components/ListComponent/ListComponent.ts

```diff
@@ -0,0 +1,36 @@
+import IComponent from '../IComponent/IComponent';
+
+import IComponentDataWithType from '../../Interfaces/interfaces';
+
+import ListComponentTemplate from './ListComponent.hbs';
+import ListComponentData from './ListComponentData';
+import './ListComponent.css';
+
+class ListComponent extends IComponent {
+    private items: IComponentDataWithType[];
+
+    get getElement(): HTMLElement {

```

Comment:
```suggestion
    get element(): HTMLElement {
```

`const listEl = list.element;`

Reply:
или
```suggestion
    getElement(): HTMLElement {
```

`const listEl = list.getElement();`